config.js

```diff
@@ -111,6 +111,8 @@ const JOB_QUEUE_TTL_SECONDS_SYNC = pe.TTL_KUE_JOBS_SYNC
 // set time interval for enableQueueStatsActivityLogs
 const queueStatsActivityLogsInterval = 60000;
 
+const DEFAULT_LIMIT = pe.DEFAULT_LIMIT || 10000;
```

Comment:
Can we make this `GET_REQUEST_DEFAULT_LIMIT`? Otherwise it feels ambiguous, confusing.